jta-and-hibernate-standalone/src/test/java/org/jboss/narayana/quickstart/jta/TestCase.java

```diff
@@ -166,6 +171,9 @@ public void testCrashBeforeCommit() throws Exception {
             transactionManager.commit();
             fail("QuickstartRuntimeException expected");
         } catch (Throwable t) {
+            if(!(t.getCause() instanceof QuickstartRuntimeException)) {
```

Comment:
I think we should change to doing this with a message in the assertTrue such as:

`assertTrue("Cause was: " + t.getClass() + ":" + t.getMessage(), t.getCause() instanceof QuickstartRuntimeException);`